### ISSUE TITLE
Does the order of prefix tags and N terminal modifications matter?

### ISSUE THREAD

**Issue by rfellers** [topdownproteomics/proteoform-nomenclature-standard#28]
I know that this is valid from the manuscript:
`[Unimod]+[1]-S[21]GRGK...`

Is this also valid ProForma?
`[1]-[Unimod]+S[21]GRGK...`

**Comment by veitveit** [comment]
I would say no. Definition of the key type should always come first.

**Comment by acesnik** [comment]
Good question! I agree with @veitveit. It's valid syntax (computer readable), but for human readability, I think we should always have the key definition to be first, since it pertains to the N-terminal modification tag.

**Comment by rleduc42** [comment]
I agree with veitveit and acesnik.

**Comment by rfellers** [comment]
Excellent, sounds like everyone is in agreement. I guess that means it is adopted? Is there a living standard document that will incorporate this clarification? Either way, I can now continue with the parser development, thanks!

**Comment by acesnik** [comment]
Yes, I think we should adopt this as an addition to Rule 6, the one that talks about prefix tags. Here is the [living document](https://github.com/topdownproteomics/ProteoformNomenclatureStandard/blob/master/Standard.md). Our first change after publishing the initial standard!
